### DISCUSSION TITLE
entity type in WikiKG90Mv2

### DISCUSSION THREAD

**Discussion by YueFan1014** [snap-stanford/ogb#355] [WikiKG90M-LSC]
Hello! May I ask whether data values could be the entities in the dataset? For example, 12-year-old, 8/1/2022, 80kg. Thanks!

**Comment by weihua916** [comment]
Hello! This is a great question!

In our dataset, we only consider entities that are of the form Q** (e.g., "Barack Obama" [Q76](https://www.wikidata.org/wiki/Q76)) and relations that are of the form P** (e.g., "instance of" [P31](https://www.wikidata.org/wiki/Property:P31)).

In the case of "Barack Obama" ([Q76](https://www.wikidata.org/wiki/Q76)), its "date of birth" ([P569](https://www.wikidata.org/wiki/Property:P31)) is "4 August 1961", but since "4 August 1961" is not in the form of Q**, we did not include `(Q76, P569, "4 August 1961")` as a triple.

**Reply by YueFan1014** [reply]
OK, thanks for your reply!
